integration-cli/docker_cli_rmi_test.go

```diff
@@ -152,7 +152,7 @@ func (s *DockerSuite) TestRmiImageIDForceWithRunningContainersAndMultipleTags(c
 
 	out, _, err := dockerCmdWithError("rmi", "-f", imgID)
 	// rmi -f should not delete image with running containers
-	c.Assert(err, checker.NotNil)
+	c.Assert(err, checker.IsNil)
```

Comment:
I don't think this is correct; also this repository tests using a 17.06 cli, if docker internal tests use a different version of the client, tests should be updated there.

Probably related: https://github.com/docker/cli/pull/418, https://github.com/moby/moby/pull/34528, and https://github.com/docker/cli/issues/394

/cc @dnephin

Reply:
I vote that if this is the case and that if this is purely a CLI test than we should remove the assertions for the exit status altogether from this repo.